File: tms-angular2-master/src/app/core/models/filterUser.ts
export class FilterUser {
    UserID: string[];
    GroupID: string[];
    Active: boolean[];

    constructor( UserID:string[],GroupID: string[],Active: boolean[]){
            this.UserID = UserID;
            this.GroupID = GroupID;
            this.Active = Active;
    }
}

File: tms-webapi-master/TMS.Common/ViewModels/FilterUser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TMS.Common.ViewModels
{
    public class FilterUser
    {
        public string[] UserID { set; get; }
        public string[] GroupID { set; get; }
        public bool[] Active { set; get; }
    }
}
